libs/labelbox/src/labelbox/client.py

```diff
@@ -1184,26 +1254,43 @@ def create_ontology(self, name, normalized, media_type=None) -> Ontology:
             name (str): Name of the ontology
             normalized (dict): A normalized ontology payload. See above for details.
             media_type (MediaType or None): Media type of a new ontology
+            ontology_kind (OntologyKind or None): set to OntologyKind.ModelEvaluation if the ontology is for chat evaluation,
+                leave as None otherwise.
+
         Returns:
             The created Ontology
+
+        NOTE caller of this method is expected to set media_type to Conversational if ontology_kind is ModelEvaluation
```

Comment:
I understand having the separate arguments for media_type and ontology_kind, but could we solve this by having ontology_kind read first before media_type? Like: If ontology_kind is set to ModelEvaluation, media_type is set to conversational, otherwise media_type is just the arg.

Reply:
Also, if you disagree with that comment and this note remains relevant, can we move it to the docstring? (But I think it's better to refactor and delete it)

Reply:
Let me ping me to understand and discuss your concern better